get_images.py

```diff
@@ -95,27 +94,40 @@ def get_image(lat: float, lon: float, heading: int = 0, fov: int = 120, size: st
     else:
         return None, None
         
+def get_images(lat: float, lon: float, num_images: int = 1, show_image: bool = False):
+    if num_images < 1:
+        raise ValueError("Number of images must be greater than 0.")
+    if num_images > 5:
+        raise ValueError("Number of images must be less than or equal to 5.")
+    headings = [0]
+    if num_images > 1:
+        headings = [heading for heading in range(0, 360, 360 // num_images)]
+    print(f"Generating images for headings: {headings}")
+    for heading in headings:
+        output_dir = pathlib.Path(IMAGES_DIR)
+        image, metadata = get_image(lat, lon, heading=heading)
+        # Save image to output dirs
+        if image:
+            # Get remove dots from lat and lon
+            lat_str = str(lat).replace(".", "dot")
+            lon_str = str(lon).replace(".", "dot")
+            output_file = output_dir / f"streetview_{metadata['date']}_{lat_str}_{lon_str}_{heading}.jpg"
+            with open(output_file, "wb") as file:
+                file.write(image)
+            print(f"Image saved to {output_file}")
+        else:
+            print("Failed to download image.")
+            exit()
```

Comment:
Maybe raise an exception, so the caller can decided to quit or keep going.

Reply:
added